x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/components/hosts_table.tsx

```diff
@@ -40,9 +40,14 @@ export const HostsTable = () => {
 
   useEffect(() => {
     if (!loading && !hostCountLoading) {
-      onPageReady();
+      onPageReady({
+        customMetrics: {
+          key1: 'hostsCount',
+          value1: count,
```

Comment:
@crespocarlos Do you think including `limit` parm could be helpful?

Reply:
yes. this info could help us decide what to do with the host limit in the future. The host limit also affects the overall performance

Reply:
https://github.com/elastic/kibana/pull/205607/commits/2e0b343336c38fb11940b9cc2768f55499147068